streampark-console/streampark-console-service/src/main/java/org/apache/streampark/console/core/service/impl/VariableServiceImpl.java

```diff
@@ -110,11 +112,40 @@ public IPage<Application> dependAppsPage(Variable variable, RestRequest request)
     page.setTotal(applications.size());
     int fromIndex = (request.getPageNum() - 1) * request.getPageSize();
```

Comment:
oh no,
There is a simpler way to implement paging, it is not recommended to set pageNo and pageSize, also not recommended to use subList

Reply:
Because the `org.apache.streampark.console.core.service.impl.VariableServiceImpl#getDependApplicationsByCode` method process data by code instead of sql.

Reply:
> Because the `org.apache.streampark.console.core.service.impl.VariableServiceImpl#getDependApplicationsByCode` method process data by code instead of sql.

get it